src/impl_x86__base_implementation.inl

```diff
@@ -695,11 +699,12 @@ X86Microarchitecture GetX86Microarchitecture(const X86Info* info) {
         return AMD_PUMA;
       case CPUID(0x17, 0x01):
       case CPUID(0x17, 0x11):
-      case CPUID(0x17, 0x18):
       case CPUID(0x17, 0x20):
+      case CPUID(0x18, 0x00):
         // https://en.wikichip.org/wiki/amd/microarchitectures/zen
         return AMD_ZEN;
       case CPUID(0x17, 0x08):
+      case CPUID(0x17, 0x18):
```

Comment:
you added detection for Picasso (ZEN+) but removed Banded Kestrel (ZEN), thus users that have Banded Kestrel will get incorrect information that is not supported by the processor

Reply:
I moved that cpuid according to wikichip, since they have the same cpuid `CPUID(0x17, 0x18)` it should be ZEN or ZEN+ not both. Do I leave ZEN as it is more general?

Reply:
yup, you should leave ZEN